docker/pull-site-and-start.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+HERE=$(dirname "$0")
+
+git clone --branch $1 $2 $3
```

Comment:
You can use `--single-branch` here maybe too?

Reply:
ive decided to use `--depth 1` instead, apparently this is better for bandwidth, see:
https://stackoverflow.com/questions/1778088/how-do-i-clone-a-single-branch-in-git?noredirect=1&lq=1